en/1-prep/2-chain-spec.md

```diff
@@ -9,7 +9,7 @@ for your relay chain network. You can use one that is included with this worksho
 
 Whichever chain spec file you choose to use we will refer to the file simply as `chain-spec.json`
 in the instructions below. You will need to supply the proper path to the chain spec you are using.
-These *conventionally* live in a `/res` folder that is published in your node's codebase for others to use. As an example:
+These _conventionally_ live in a `/res` folder that is published in your node's codebase for others to use. As an example:
```

Comment:
fyi, Using `*xxx*`, and `_xxx_` have the same effect in markdown.
More info: https://guides.github.com/features/mastering-markdown/ (search for "Emphasis")